test-cases/features/add-new-dc.yaml

```diff
@@ -0,0 +1,9 @@
+n_db_nodes: 3 0  # make n_db_nodes configured as multi-dc with last dc set to 0 (so later easily new node can be added)
+region_name: 'eu-west-1 eu-west-2'
+n_loaders: 1
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.xlarge'
+seeds_num: 3
```

Comment:
why do you have 3 seed nodes?

Reply:
I didn't pay attention to that. How much it should be?